packages/api/models/granules.js

```diff
@@ -448,6 +457,34 @@ class Granule extends Manager {
     return super.delete({ granuleId: granule.granuleId });
   }
 
+  /**
+   * Delete a published granule.
```

Comment:
Nit - this should probably note that unpublishes *and* deletes in the description.     Possibly consider making the function name something like deleteAndUnpublishGranule (but that doesn't feel great). 

This is fine as-is, take/leave the above.